File: ASP.NET-projekat/BerixLibrary/Application/DTOs/ShippingMethods/ShippingMethodDTO.cs
using Application.DTOs.Orders;
using Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application.DTOs.ShippingMethods
{
    public class ShippingMethodDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public decimal Cost { get; set; }
        virtual public ICollection<OrderDTO>? Orders { get; set; } = new List<OrderDTO>();
    }
}

File: src/tsBusinessLayer/dtos/ShippingMethodDTO.ts
import {IShippingMethod} from "../interfaces/IShippingMethod"

export class ShippingMethodDTO implements IShippingMethod{
    public id:number;
    public Name:String;
    public Cost:number;

    constructor(id:number, name: String, cost: number){
        this.id = id;
        this.Name = name;
        this.Cost = cost;
    }
}